build.gradle.kts

```diff
@@ -107,7 +106,7 @@ tasks.jacocoTestReport {
         xml.required.set(true)
         csv.required.set(false)
     }
-    executionData.setFrom(fileTree(projectDir).include("*.exec"))
+    executionData.setFrom(fileTree(projectDir).include("**/*.exec"))
```

Comment:
Es entzieht sich meiner Kenntnis, wo sich die .exec Dateien befinden. Die CI versucht auf jeden Fall auch noch Dateien der Form ./*.exec zu kopieren. Die sollten da inbegriffen sein, oder?
Ansonsten würde **.exec die auch mit einschließen.

Reply:
**/*.exec heißt: Jede .exec Datei die in einem beliebigen Subfolder liegt.
*.exec heißt: jede exec Datei die im Folder selbst liegt.

Einige Files liegen aber in Subfoldern.